envs/example/defaults-2.0.yml

```diff
@@ -255,6 +255,7 @@ neutron:
   tenant_nameservers:
     - 8.8.4.4
     - 8.8.8.8
+  enable_external_interface: True
```

Comment:
Instead of adding it to defaults here, we need to add it to the allinone example.